### ISSUE TITLE
can u add users page?

### ISSUE THREAD

**Issue by aniel300** [antebrl/IPTV-Restream#43]
can u add users page? with cxn, expiry, etc and add xc api support?

**Comment by antebrl** [comment]
Thanks for the idea. 
At the moment, this application is primarely used for (friend) groups, where one user can share his iptv channels with the others. So only this user would need such a users page. It would be an admin page.
Such an admin page is planned though. I'll include your requested features in it. 
But there are some requirements like authentication etc. to make such an page. Therefore and because it is not urgently needed, it will probably take some time before I'll implement this.

**Comment by aniel300** [comment]
yes that is what i was asking. a server admin page for the server admin. one for users (with cxn, expiry, etc and add xc api support) and another for streams (streams/channels, groups, etc management)
